test/swift/apps/experimental/ViewController.swift

```diff
@@ -17,6 +18,9 @@ final class ViewController: UITableViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
+    // swiftlint:disable:next force_unwrapping
+    let userDefaults = UserDefaults(suiteName: kPersistenceKey)!
```

Comment:
can we just return if created `UserDefaults` is null as opposed to force unwrapping?

Reply:
For this demo we should use the standard user defaults, which is what most users would want to reach for first anyway:

```suggestion
    let userDefaults = UserDefaults.standard
```

Reply:
Yes, good point